client/js/components/statement/splitStatement/SplitStatementView.vue

```diff
@@ -746,27 +802,32 @@ export default {
       if (this.segments.length > 0) {
         if (window.dpconfirm(Translator.trans('statement.split.complete.confirm'))) {
           this.setProperty({ prop: 'isBusy', val: true })
-          try {
-            // Set data with html not only charStart and charEnd
-            const ranges = this.prosemirror.keyAccess.rangeTrackerKey.getState(this.prosemirror.view.state)
-            const segmentsWithText = this.segments
-              .filter(segment => !!ranges[segment.id])
-              .map(segment => {
-                return {
-                  ...segment,
-                  text: ranges[segment.id].text
-                }
-              })
-            this.setProperty({ prop: 'segmentsWithText', val: segmentsWithText })
-            const currentStatementText = this.prosemirror.getContent(this.prosemirror.view.state)
-            this.setProperty({ prop: 'statementText', val: currentStatementText })
-            this.saveSegmentsFinal()
-              .then(() => this.setProperty({ prop: 'isBusy', val: false }))
-          } catch (err) {
-            console.error('An error occurred:', err)
-            dplan.notify.error(Translator.trans('error.api.generic'))
-            this.setProperty({ prop: 'isBusy', val: false })
-          }
+          /*
+           * Try {
+           *  Set data with html not only charStart and charEnd
+           */
+          const ranges = this.prosemirror.keyAccess.rangeTrackerKey.getState(this.prosemirror.view.state)
+          const segmentsWithText = this.segments
+            .filter(segment => !!ranges[segment.id])
+            .map(segment => {
+              return {
+                ...segment,
+                text: ranges[segment.id].text
+              }
+            })
+          console.log('segmentsWithText: ', segmentsWithText)
+          this.setProperty({ prop: 'segmentsWithText', val: segmentsWithText })
+          const currentStatementText = this.prosemirror.getContent(this.prosemirror.view.state)
+          this.setProperty({ prop: 'statementText', val: currentStatementText })
+          /*
+           *  This.saveSegmentsFinal()
+           *  .then(() => this.setProperty({ prop: 'isBusy', val: false }))
+           *  } catch (err) {
+           *  console.error('An error occurred:', err)
+           *  dplan.notify.error(Translator.trans('error.api.generic'))
+           *  this.setProperty({ prop: 'isBusy', val: false })
+           *  }
+           */
```

Comment:
can this be removed?

Reply:
it is currently not possible to save the segments without charStart/charEnd positions, i get an error from the backend. But for now, that is ok, we will leave it as it is. This can be handled/discussed later with the backend.

i removed comment markers here https://github.com/demos-europe/demosplan-core/pull/4473/commits/ba741571752f908f15f4d656e8396d80c6a42ab2